docs/pages/management/guides.mdx

```diff
@@ -10,6 +10,8 @@ documented elsewhere:
 
   - [EC2 tags as Teleport Node labels](./guides/ec2-tags.mdx). How to set up
     Teleport Node labels based on EC2 tags.
+  - [GCP tags and labels as Teleport Node labels](./guides/gcp-tags.mdx). How
```

Comment:
We don't use the term "Node" anymore in the docs since we've had issues with ambiguity. Does this refer to labels for Teleport-protected servers?